etc/testing/travis.sh

```diff
@@ -64,6 +64,83 @@ if [[ "$BUCKET" == "MISC" ]]; then
         make lint enterprise-code-checkin-test docker-build test-pfs-server \
             test-pfs-cmds test-deploy-cmds test-libs test-admin test-s3
     fi
+elif [[ "$BUCKET" == "EXAMPLES" ]]; then
+    echo "Running the example test suite"
```

Comment:
In general I think this approach is great, though I wonder if it might make sense to put these tests in their own bash files, in case we want to run them locally